src/index.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App/App';
+import './CSS/index.scss';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import App from './App/Components/App';
 import reportWebVitals from './reportWebVitals';
+import AllReducers from './Redux/reducers/index';
 // import * as serviceWorkerRegistration from './serviceWorkerRegistration';
```

Comment:
- [ ]  [REQUIRED] Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history